StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/STPElementsSession.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+@_spi(STP) import StripePayments
```

Comment:
Why is this needed? I don't see any changes to the file.

Reply:
Oh some file was moved?

Reply:
Yeah I think STPElementsSession should be in the StripePaymentSheet module.